source/_integrations/hassio.markdown

```diff
@@ -116,6 +116,7 @@ Create a full backup.
 | ---------------------- | -------- | ----------- |
 | `name` | yes | Name of the backup file. Default is current date and time
 | `password` | yes | Optional password for backup
+| `compressed` | yes | `false` to create uncompressed backups
```

Comment:
I can't find the Core PR that adds this, nor can I find this capability in the current Core development branch.

Could you point me to it?

Reply:
Ok, looks like this was not included already in core itself, only in hassio.

Done it here https://github.com/home-assistant/core/pull/70819

Let me rebase this against `next` then.

Reply:
Uff, I'm more used to gitlab, didn't find the change-base button till now... Well too late, opened #22533 instead